op-acceptance-tests/tests/base/advance_test.go

```diff
@@ -29,3 +30,11 @@ func TestCLAdvance(gt *testing.T) {
 		return new_num > num
 	}, 30*time.Second, waitTime)
 }
+
+// TestCLAdvance_Supernode runs the same advance test using the supernode L2CL implementation.
+func TestCLAdvance_Supernode(t *testing.T) {
+	prev := os.Getenv("DEVSTACK_L2CL_KIND")
+	_ = os.Setenv("DEVSTACK_L2CL_KIND", "supernode")
+	defer func() { _ = os.Setenv("DEVSTACK_L2CL_KIND", prev) }()
```

Comment:
I like how you're wrapped the existing test!

Note that there's a built-in for the temporary environment variable setting.
It'll make our implementation shorter and probably also safer for parallel runs.

```suggestion
	t.Setenv("DEVSTACK_L2CL_KIND", "supernode")
```

Reply:
Ah thank you for this, much simpler! I'll replace that.

And regarding your top level comment "Are there no more AT's we could add to test SuperNode?" : yes I think I need to enable more, but I wanted to start with a basic "does it serve as a CL" and get the pattern. There are aspects of node operation that aren't expected to work (namely P2P) but I still need to do a pass and enable more tests now that I have your confirmation that it looks alright.